integration/src/main/java/scripts/utils/GcsBucketAccessTester.java

```diff
@@ -104,45 +165,95 @@ public void checkAccess(TestUserSpecification testUser, @Nullable ControlledReso
 
     testClient = ClientTestUtils.getGcpStorageClient(testUser, projectId);
     if (role == null) {
-      assertFalse(doWithOptionalWait(() -> blobCreate(testClient)), "no role cannot create");
-      assertFalse(doWithOptionalWait(this::blobRead), "no role cannot read");
-      assertFalse(doWithOptionalWait(this::blobUpdate), "no role cannot update");
-      assertFalse(doWithOptionalWait(this::blobDelete), "no role cannot delete");
-      assertFalse(doWithOptionalWait(this::bucketGet), "no role cannot bucket get");
-      assertFalse(doWithOptionalWait(this::bucketDelete), "no role cannot bucket delete");
-      return;
+      if (doNoWait(() -> blobCreate(testClient))) {
```

Comment:
can you add some comment to explain why these use `doNoWait` regardless of what the `wait` param is?

Reply:
I added a comment at the top of the method, since it applies to all of the cases.